helm/fmtok8s-conference-chart/templates/ingress.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.grafana.enabled }}
```

Comment:
@mcruzdev why is this related to grafana? This is the ingress for the frontend so it shouldn't be related to grafana